src/main/java/io/github/sefiraat/networks/slimefun/network/NetworkQuantumStorage.java

```diff
@@ -353,6 +408,10 @@ public int getMaxAmount() {
         return maxAmount;
     }
 
+    public boolean getSupportsCustomMaxAmount() {
```

Comment:
You don't need to call this "get..." it should just be supportsCustomMaxAmount because it's a Boolean